MarsQACompetitionTask/POMTests/POM.cs

```diff
@@ -0,0 +1,103 @@
+﻿using MarsQACompetitionTask.Pages;
+using MarsQACompetitionTask.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+//using OpenQA.Selenium.DevTools.V114.Profiler;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MarsQACompetitionTask.POMTests
+{
+    [TestFixture]
+    public class POM : CommonDriver
+    {
+        /*public IWebDriver driver;
+        public SignIn SignInObj;
+        public Education EducationObj;
+        public Certifications CertificationObj;*/
+        
+
+        [Test, Order(1), Description("Create Record")]
+        public void CreateEducationTests()
+        {
+            SignInObj.SigninStep();
```

Comment:
You need to think about state!

example before adding language you need to make sure there a no existing languages as this could result in false positive test pass
you will need to think about this for all the scenarios

the state is say that you have a test that is adding a language english, the second time you run this test you should delete any existing language so that the state is reset, this will guarantee that your test does not produce false negatives and passes always
you can assume that test execution order is not always guaranteed
imagine if you run the tests  in parallel mode the all tests will run at the same time
so you need to also make sure you do not wipe out other tests data
hence why when you remove data the best thing to do is delete only data you added for the test to be correct
e.g test 1 added english and french as language test 2 added hindi then when you reset data you already know what language you are adding for each test
there for on reset of the two test you should delete english and french for test 1 and hindi for test 2

Reply:
this is still not done can you please so this